src/sdk/AvatarConfig.ts

```diff
@@ -43,19 +45,49 @@ export class AvatarConfig {
     }
 
     /**
-     * Sets the background color.
+     * Gets the background color.
      */
     public get backgroundColor(): string {
         return this.privBackgroundColor;
     }
 
     /**
-     * Gets the background color.
+     * Sets the background color.
      */
     public set backgroundColor(value: string) {
         this.privBackgroundColor = value;
     }
 
+    /**
+     * Gets the background image.
+     */
+    public get backgroundImage(): URL {
+        return this.privBackgroundImage;
+    }
+
+    /**
+     * Sets the background image.
+     * @param {URL} value - The background image.
+     */
+    public set backgroundImage(value: URL) {
+        this.privBackgroundImage = value;
+    }
+
+    public get remoteICEServers(): RTCIceServer[] {
+        return this.privRemoteICEServers;
+    }
+
+    /**
+     * Sets the remote ICE servers.
+     * @remarks Normally, the ICE servers are gathered from the PeerConnection,
+     * set this property to override the ICE servers. E.g., the ICE servers are
+     * different in client and server side.
+     * @param {RTCIceServer[]} value - The remote ICE servers.
+     */
+    public set remoteICEServers(value: RTCIceServer[]) {
```

Comment:
Can you rename remoteICEServers to remoteIceServers to have consistent naming with others?